custom_components/ideenergy/sensor.py

```diff
@@ -152,6 +138,49 @@ async def async_get_last_state(self):
 
         return {}
 
+class InstantConsumption(RestoreEntity, IDeEntity, SensorEntity):
```

Comment:
Check current sensors in main please, I have changed some base classes, may be you need to update this.